spec/requests/todos_spec.rb

```diff
@@ -0,0 +1,63 @@
+require "rails_helper"
+
+RSpec.describe "API V1 ToDo", type: :request do
```

Comment:
I might also use `type: controller` here since we are testing the todo controller.